haxe_libraries/formatter.hxml

```diff
@@ -1,4 +1,4 @@
 # @install: lix --silent download "haxelib:/formatter#1.12.0" into formatter/1.12.0/haxelib
-# @run: haxelib run-dir formatter ${HAXE_LIBCACHE}/formatter/1.12.0/haxelib
+# @run: haxelib run-dir formatter "${HAXE_LIBCACHE}/formatter/1.12.0/haxelib"
```

Comment:
Is this a manual change? These files are auto-generated by lix, so this would be undone the next time we update the libs. It has to be fixed there.

Reply:
Then it also needs to be fixed in lix directly.

Reply:
@Gama11 Is that the location where quotes need to be added? https://github.com/lix-pm/lix.client/blob/d026272935ac1d26d72d3b80f0f0e14222b445a5/src/lix/client/Archives.hx#L237

Reply:
Possibly? I'm roughly as familiar with the lix sources as you. :D